frontend/codegen.yml

```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: "https://graphql.preprod.crownlabs.polito.it/"
+schema: schema.graphql
```

Comment:
Is this change intentional? If so, would add a comment about the fact that the schema needs to be manually downloaded.